client/webserver/site/src/js/markets.ts

```diff
@@ -802,6 +808,22 @@ export default class MarketsPage extends BasePage {
       Doc.bind(bttn, 'click', () => this.candleDurationSelected(dur))
       page.durBttnBox.appendChild(bttn)
     }
+
+    // Check if we already have the fiveMinBin candles cache, if not, request
+    // for it first since we use it to determine the 24hour high/low in
+    // this.setHighLow() as a fallback if there is no data for 24hour high/low
+    // in spot.
+    const lastCandleDur = State.fetchLocal(State.lastCandleDurationLK)
+    const cache = this.market?.candleCaches[fiveMinBinKey]
+    if (!cache && lastCandleDur !== fiveMinBinKey) {
+      this.candleDur = fiveMinBinKey
+      this.requestCandles()
+    }
```

Comment:
This seems like the wrong place to do this. `setCandleDurBttns` is called in `setMarket` before `candleDurationSelected -> loadCandles` below, so `candleCaches` will always be empty.